Automattic/wp-calypso
# Sensei Onboarding - Wire up Course Theme style variations with Sensei Flow

#### Proposed Changes

* Wire up Course theme style variations with the Sensei Tailored Flow. Sets the style variation that user selects in the Sensei Setup screen.

#### Testing Instructions



* Start the Sensei Tailored Flow at `https://calypso.localhost:3000/setup/sensei`
* Select the desired style on the Sensei Setup screen. Choose anything besides the default one.
* Go through the rest of the steps and get to the Sensei Launchpad screen.
* Confirm the created site is in the style variation that you selected in the Sensei Setup screen earlier.

#### Pre-merge Checklist





Related to https://github.com/Automattic/sensei-pro/issues/1866
